quic/transport/timeout.nim

```diff
@@ -32,3 +35,4 @@ proc set*(timeout: Timeout, duration: Duration) =
 
 proc expired*(timeout: Timeout) {.async.} =
   await timeout.expired.wait()
+  trace "expired"
```

Comment:
are these `trace` logs meant to stay here?

Reply:
They will be removed once I figure out how timeout work on ngtcp2